src/index.ts

```
@@ -0,0 +1,440 @@
+
+// Event: Slash command interaction
+client.on(Events.InteractionCreate, async (interaction) => {
+  if (!interaction.isChatInputCommand()) return;
+  if (!interaction.guild) return;
+
+  try {
+    switch (interaction.commandName) {
+      case "santa":
+        console.log("santa command called");
+        const role = interaction.options.getRole("role", true);
+        await santa(interaction, role as Role);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)